packages/components/src/utils/form/isWithinForm.spec.ts

```diff
@@ -0,0 +1,23 @@
+import * as getClosestHTMLElementUtils from '../dom/getClosestHTMLElement';
+import { isWithinForm } from './isWithinForm';
+
+describe('isWithinForm()', () => {
```

Comment:
Unnecessary describe block since it is redundant with the name ofthe test suite.